src/cloud_auth.ts

```diff
@@ -53,16 +54,17 @@ export class CloudAuth implements Authenticator {
         if (!config['cmd-path']) {
             throw new Error('Token is expired!');
         }
-        const args = config['cmd-args'];
+        let args: string[] = [];
+        const cmdargs = config['cmd-args'];
+        if (cmdargs) {
+            args = cmdargs.split(' ');
```

Comment:
This is going to break if the string is something like:

`"foo bar" baz`

Reply:
right, so I checked how the native CLI do implement the same and it doesn't handle this "corner-case": https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/plugin/pkg/client/auth/gcp/gcp.go#L145
So I was assuming this is safe to do the same

Reply:
updated PR to rely on `child_process`, who consumes a plain string command (not individual args) and actualy makes code simpler here.
Alternatively could use string-argv at the cost of an extra dependency.